ekip/everykid/templates/get-your-pass/game_success.html

```diff
@@ -1,24 +1,49 @@
 {% extends "base.html" %}
+{% load staticfiles %}
 
 {% block title %}Get Your Pass - Fourth Graders{% endblock title %}
 
 {% block page_id %}student-pass{% endblock page_id %}
 
-{% block background %}brown-rocks{% endblock background %}
+{% block background %}tent{% endblock background %}
 
 {% block content %}
 
-Success! You will get your voucher on the next screen. Please print your
-voucher, keep it in a safe place, and remember to take it with you on your
-trip.
+  <div class="pass-content voucher-claim">
 
-When you visit your federal lands and waters, we'd love to know where you are
-coming from.
+    <h1>You did it!<br/>Claim your voucher</h1>
 
-<form method="POST">
-    {% csrf_token %}
-    {{form}}
-    <button type="input" value="submit">Submit</button>
-</form>
+    <div class="voucher-claim-description">
+
+      <p>You will get your voucher on the next screen. Please print it and keep it in a safe place, and remember to bring it with you on your trip! Just tell us your zip code so that we know where you are coming from when you visit.</p>
```

Comment:
Change this text to: 
Success! You will get your voucher on the next screen. Please print your voucher, keep it in a safe place, and remember to take it with you on your trip. When you visit your federal lands and waters, we'd love to know where you are coming from.